build-ps/percona-server.spec

```diff
@@ -497,19 +501,30 @@ datadir=$(/usr/bin/my_print_defaults server mysqld | grep '^--datadir=' | sed -n
 %endif
 # We need this because we don't provide my.cnf on centos 6
 # and the default system one doesn't have info needed
-%if 0%{?rhel} < 7
-if [ $1 -eq 1 ]; then
+#%if 0%{?rhel} < 7
+#if [ $1 -eq 1 ]; then
   cnflog=$(/usr/bin/my_print_defaults mysqld|grep -c log-error)
   if [ $cnflog = 0 -a -f /etc/my.cnf ]; then
     sed -i "/^\[mysqld\]$/a log-error=/var/log/mysqld.log" /etc/my.cnf
   fi
+  sed -i "/^\[mysqld\]$/a log-error=/var/log/mysqld.log" /etc/percona-server.conf.d/my.cnf
```

Comment:
do we want this executed every time?
is the /etc/percona-server.conf.d/my.cnf correct or it should be /etc/mysql/percona-server.conf.d/my.cnf?
do we need this if we provide our own cnf file with log-error included?